cli/cli_app.go

```diff
@@ -233,7 +233,7 @@ func downloadModules(terragruntOptions *options.TerragruntOptions) error {
 			return err
 		}
 		if shouldDownload {
-			return shell.RunTerraformCommand(terragruntOptions, "get", "-update")
+			return shell.RunTerraformCommandAndIgnoreOutput(terragruntOptions, "get", "-update")
```

Comment:
I agree with you. But the goal is not only limit the quantity of output, but also not "pollute" the output of the effective target command with artefacts that are required by terragrunt.

i.e. If i do `terragrunt plan`, I am interested in the output of `terraform plan`. Any other command that are issued by `terragrunt` like `terraform init`, `terraform get` are only required to achieve the `terraform plan` so their output should not be mixed with the output of the plan and should be disabled if required.